project-base/src/Shopsys/ShopBundle/Controller/Front/ImageController.php

```diff
@@ -2,9 +2,10 @@
 
 namespace Shopsys\ShopBundle\Controller\Front;
 
+use League\Flysystem\FilesystemInterface;
 use Shopsys\FrameworkBundle\Component\Image\Config\ImageConfig;
 use Shopsys\FrameworkBundle\Component\Image\Processing\ImageGeneratorFacade;
-use Symfony\Component\HttpFoundation\BinaryFileResponse;
+use Symfony\Component\HttpFoundation\StreamedResponse;
```

Comment:
This change should be mentioned in the CM

Reply:
What should I say? We changed ImageController to work with the abstract filesystem? So it uses StreamedResponse because BinaryFileResponse could not be used?

Reply:
And you can explay why it doesn't work. Or why streamed response works insted.